src/main/java/com/github/redouane59/twitter/IAPIEventListener.java

```diff
@@ -0,0 +1,41 @@
+package com.github.redouane59.twitter;
+
+import com.github.redouane59.twitter.dto.others.TweetError;
+import com.github.redouane59.twitter.dto.tweet.TweetV2;
+
+public interface IAPIEventListener {
+  
+  /**
+   * Triggered where an error from an API is received
+   * an httpcode different from 200 is considered as an error.
+   * 
+   * @param httpCode
+   * @param json
+   */
+  void onError( int httpCode, TweetError error);

```

Comment:
Can you use [the project style](https://github.com/redouane59/twittered/blob/develop/.idea/codeStyles/GoogleStyle.xml) just to avoid some strange spaces & indentations ? Thanks :)

Reply:
yep. no pb.